ComfyUI/comfy_api_nodes/nodes_runway_text2img.py

```
@@ -0,0 +1,269 @@
+            final_response = self.get_response(
+                task_id=initial_response.id,
+                auth_kwargs={"api_key": os.environ["RUNWAY_API_KEY"]},
+                node_id=unique_id,
+                poll_timeout=poll_timeout,
+            )
+            self.validate_response(final_response)
+            
+            # Download and return the generated image
+            image_url = final_response.output[0]  # Get first output URL
+            image_tensor = download_url_to_image_tensor(image_url)
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)